sa/saro.go

```diff
@@ -1235,13 +1235,20 @@ func (ssa *SQLStorageAuthorityRO) SerialsForIncident(req *sapb.SerialsForInciden
 			return err
 		}
 
-		err = stream.Send(
-			&sapb.IncidentSerial{
-				Serial:         ism.Serial,
-				RegistrationID: ism.RegistrationID,
-				OrderID:        ism.OrderID,
-				LastNoticeSent: ism.LastNoticeSent.UnixNano(),
-			})
+		ispb := &sapb.IncidentSerial{
+			Serial: ism.Serial,
+		}
+		if ism.RegistrationID != nil {
+			ispb.RegistrationID = *ism.RegistrationID
+		}
```

Comment:
Just to confirm: it's okay for RegistrationID, OrderID, and LastNoticeSent to have their zero values when they are NULL in the DB? Should we document that in the `IncidentSerial` proto?

Reply:
Confirmed. The only (current) caller of SerialsForIncident is [admin-revoker](https://github.com/letsencrypt/boulder/blob/08017e436eac51aeffb6d930e4daaf82179bc3c4/cmd/admin-revoker/main.go#L268), and it only [uses](https://github.com/letsencrypt/boulder/blob/08017e436eac51aeffb6d930e4daaf82179bc3c4/cmd/admin-revoker/main.go#L283) the serial itself.

I've added documentation to this effect.